bindata/v4.0.0/apiservice-cabundle-controller/defaultconfig.yaml

```diff
@@ -5,3 +5,7 @@ authentication:
   disabled: true
 authorization:
   disabled: true
+leaderElection:
+  leaseDuration: "5s"
+  renewDeadline: "2s"
+  retryPeriod: "1s"
```

Comment:
before this: https://github.com/openshift/library-go/pull/305  defaults were the k8s defaults, 
retry: 2s
renew: 10s
leaseDur: 15s
Do we need them to be this short? 1s,2s,5s?  Maybe bump them back to what they were, unless we were seeing issues with them being too long before?